var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
 
     version('1.1.6', '5f13a0841a61f7fc295c514490d120d0',
             url="https://pypi.python.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz")
 
-    depends_on('python@2.4:2.8,3.3:')
+    depends_on('python@2.4:3.3.99')
```

Comment:
`2.4:3.3` is fine here. But yes, you are right, it should work for Python 3.1 and 3.2. I was confused when I added this line because the `setup.py` contains:
```python
        classifiers=[                                                           
            'Development Status :: 5 - Production/Stable',                      
            'Intended Audience :: Developers',                                  
            'License :: OSI Approved :: BSD License',                           
            'Programming Language :: Python',                                   
            'Topic :: Software Development',                                    
            'Programming Language :: Python :: 2.4',                            
            'Programming Language :: Python :: 2.5',                            
            'Programming Language :: Python :: 2.6',                            
            'Programming Language :: Python :: 2.7',                            
            'Programming Language :: Python :: 3.3',                            
            'Programming Language :: Python :: 3.4',                            
            'Programming Language :: Python :: 3.5',                            
            ],                                     
```
At this point I'm not sure what this means anymore. I thought it meant that Python 3.0 - 3.2 were not supported, but I guess not.